tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -0,0 +1,129 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import RowSchema, Field
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T2
+
+
+t2 = RowSchema(
```

Comment:
@elipe17 this is the list of fields for TANF T2 records:

```
T2col_list = ['RecordType','RPT_MONTH_YEAR', 'CASE_NUMBER','FAMILY_AFFILIATION','NONCUSTODIAL_PARENT', 'DATE_OF_BIRTH','SSN','RACE_HISPANIC',
            'RACE_AMER_INDIAN','RACE_ASIAN',  'RACE_BLACK',  'RACE_HAWAIIAN', 'RACE_WHITE',
            'GENDER', 'FED_OASDI_PROGRAM',  'FED_DISABILITY_STATUS', 'DISABLED_TITLE_XIVAPDT', 'AID_AGED_BLIND',                
            'RECEIVE_SSI','MARITAL_STATUS', 'RELATIONSHIP_HOH',  'PARENT_MINOR_CHILD',  'NEEDS_PREGNANT_WOMAN',          
            'EDUCATION_LEVEL', 'CITIZENSHIP_STATUS', 'COOPERATION_CHILD_SUPPORT', 'MONTHS_FED_TIME_LIMIT',  'MONTHS_STATE_TIME_LIMIT',       
            'CURRENT_MONTH_STATE_EXEMPT', 'EMPLOYMENT_STATUS',  'WORK_ELIGIBLE_INDICATOR',   'WORK_PART_STATUS',  'UNSUB_EMPLOYMENT',              
            'SUB_PRIVATE_EMPLOYMENT',  'SUB_PUBLIC_EMPLOYMENT',  'WORK_EXPERIENCE_HOP', 'WORK_EXPERIENCE_EA', 'WORK_EXPERIENCE_HOL',           
            'OJT',  'JOB_SEARCH_HOP',  'JOB_SEARCH_EA', 'JOB_SEARCH_HOL',  'COMM_SERVICES_HOP',             
            'COMM_SERVICES_EA', 'COMM_SERVICES_HOL', 'VOCATIONAL_ED_TRAINING_HOP', 'VOCATIONAL_ED_TRAINING_EA', 'VOCATIONAL_ED_TRAINING_HOL',    
            'JOB_SKILLS_TRAINING_HOP',  'JOB_SKILLS_TRAINING_EA',  'JOB_SKILLS_TRAINING_HOL',  'ED_NO_HIGH_SCHOOL_DIPL_HOP',  'ED_NO_HIGH_SCHOOL_DIPL_EA',     
            'ED_NO_HIGH_SCHOOL_DIPL_HOL', 'SCHOOL_ATTENDENCE_HOP', 'SCHOOL_ATTENDENCE_EA',   'SCHOOL_ATTENDENCE_HOL',   'PROVIDE_CC_HOP',                
            'PROVIDE_CC_EA',  'PROVIDE_CC_HOL',  'OTHER_WORK_ACTIVITIES',  'DEEMED_HOURS_FOR_OVERALL',  'DEEMED_HOURS_FOR_TWO_PARENT',   
            'EARNED_INCOME', 'UNEARNED_INCOME_TAX_CREDIT', 'UNEARNED_SOCIAL_SECURITY', 'UNEARNED_SSI', 'UNEARNED_WORKERS_COMP',         
            'OTHER_UNEARNED_INCOME']
```
also located [here](https://gorafttech.sharepoint.com/:u:/r/sites/TDRSResearchDesign/Shared%20Documents/Dev/dev/Parsing/py/[TANF_Parsing.py](https://gorafttech.sharepoint.com/:u:/r/sites/TDRSResearchDesign/Shared%20Documents/Dev/dev/Parsing/py/TANF_Parsing.py?csf=1&web=1&e=qFRqo9)?csf=1&web=1&e=qFRqo9)